nuget/spec/fixtures/github/contents_dotnet_directory_packages_props.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Directory.Packages.props",
```

Comment:
For anyone else reviewing, this response is for the file https://github.com/TobiasLaving/nuget-directory-packages-props-example/blob/main/Directory.Packages.props